### PR TITLE
Wiley/refactoring

### PR DESCRIPTION
**Pull request by wileyw**

### CODE REVIEW

**Review by mswang12 — COMMENTED** [review]
Looks good! Have you verified that the code still works?

**Comment by wileyw** [comment]
I haven't validated it yet.

File: d_net.py [hunk 1]
```diff
@@ -16,51 +16,40 @@
...
-    [1024, 512, 1],
-]
+SCALE_KERNEL_SIZES_D = [[3], [3, 3, 3], [5, 5, 5], [7, 7, 5, 5]]
+# layer sizes for each fully-connected layer of each scale network
+# in the discriminator model layer connecting conv to fully-connected is dynamically
+# generated when creating the model
+SCALE_FC_LAYER_SIZES_D = [[512, 256, 1], [1024, 512, 1], [1024, 512, 1], [1024, 512, 1]]
 
 
+# TODO: Not used, remove?
```
Comment (jinac) [inline d_net.py]: DiscriminatorModel is composed of multiple DScaleNet. This shouldn't be removed.

File: d_net.py [hunk 2]
```diff
@@ -16,51 +16,40 @@
...
-]
-# layer sizes for each fully-connected layer of each scale network in the discriminator model
-# layer connecting conv to fully-connected is dynamically generated when creating the model
-SCALE_FC_LAYER_SIZES_D = [
-    [512, 256, 1],
-    [1024, 512, 1],
-    [1024, 512, 1],
-    [1024, 512, 1],
-]
+SCALE_KERNEL_SIZES_D = [[3], [3, 3, 3], [5, 5, 5], [7, 7, 5, 5]]
```
Comment (jinac) [inline d_net.py]: I don't have too strong of an opinion on the formatting of this, but I feel the old way was more readable.

File: example.py [hunk 3]
```diff
@@ -14,52 +13,56 @@ def __len__(self):
         return 1
 
     def __getitem__(self, idx):
-        image_path = '/Users/michael/repo/VideoGAN/Ms_Pacman/Test/0000/00070.png'
+        image_path = "/Users/michael/repo/VideoGAN/Ms_Pacman/Test/0000/00070.png"
```
Comment (jinac) [inline example.py]: Can we not do this absolute filepaths?
Or are will we remove this file altogether? Is it needed?

File: data_util.py [hunk 4]
```diff
@@ -33,26 +33,29 @@ def get_full_clips(data_dir, hist_len, num_clips, num_rec_out=1):
...
-                      FULL_HEIGHT,
-                      FULL_WIDTH,
-                      (3 * (hist_len + num_rec_out))])
+    clips = np.empty(
+        [num_clips, FULL_HEIGHT, FULL_WIDTH, (3 * (hist_len + num_rec_out))]
+    )
 
     # get num_clips random episodes
-    ep_dirs = np.random.choice(glob(os.path.join(data_dir, '*')), num_clips)
+    ep_dirs = np.random.choice(glob(os.path.join(data_dir, "*")), num_clips)
```
Comment (jinac) [inline data_util.py:41]: I thought single quotes was more "pythonic". Don't really care which way we do it as long as we're consistent.

File: process.py [hunk 5]
```diff
@@ -1,47 +1,38 @@
...
 
-    if not os.path.exists('output'):
-        os.makedirs('output')
-    scipy.misc.imsave('output/{}_{:05d}.jpg'.format(prefix, iteration), grid)
+    if not os.path.exists("output"):
+        os.makedirs("output")
+    scipy.misc.imsave("output/{}_{:05d}.jpg".format(prefix, iteration), grid)
 
 
 def get_emoji_loader(emoji_type):
```
Comment (jinac) [inline process.py]: I don't think we need this function.

File: process.py [hunk 6]
```diff
@@ -53,67 +44,84 @@ def get_emoji_loader(emoji_type):
...
-            conv_layer_fms_list=SCALE_CONV_FSM_D,
-            scale_fc_layer_sizes_list=SCALE_FC_LAYER_SIZES_D)
-        video_d_net.type(dtype)
-
-        video_g_net = vanilla_gan.video_gan.VideoGANGenerator()
-        video_g_net.type(dtype)
-
-        video_d_optimizer = optim.Adam(video_d_net.parameters(), lr=0.0001)
-        video_g_optimizer = optim.Adam(video_g_net.parameters(), lr=0.0001)
+    SCALE_CONV_FSM_D = [
```
Comment (jinac) [inline process.py]: Since we have a config python file, should we move this here and import it?

**Review by jinac — COMMENTED** [review]
Just a couple of comments

File: d_net.py (hunk 2 above)
Comment (wileyw) [reply d_net.py]: True, it was auto formatter

**Review by wileyw — COMMENTED** [review]
Seems working. Still need to fix the conflict. And other small format things.

**Comment by wileyw** [comment]
@jinac , is it possible for you to solve the conflicts and merge?

**Comment by wileyw** [comment]
I will merge the code into master, if no objection raises in the next few hours.

**Pull request merged**